services/tenant-ui/frontend/src/components/contacts/Contacts.vue

```diff
@@ -12,48 +20,61 @@
       <Column field="contact_id" header="ID" />
     </DataTable>
   </div>
-  <Button v-if="store.state.contacts.data" class="create-contact" icon="pi pi-plus" label="Create Contact" @click="createContact"></Button>
-
-  <Dialog header="Create a new contact" v-model:visible="displayAddContact" :modal="true">
+  <Button
+    v-if="store.state.contacts.data"
+    class="create-contact"
+    icon="pi pi-plus"
+    label="Create Contact"
+    @click="createContact"
+  ></Button>
+
+  <Dialog
+    header="Create a new contact"
+    v-model:visible="displayAddContact"
+    :modal="true"
+  >
     <CreateContact @created="contactCreated" />
   </Dialog>
 </template>
 
 <script setup lang="ts">
 // Vue
-import { inject, ref, onMounted } from 'vue';
+import { inject, ref, onMounted } from "vue";
 
 // PrimeVue
-import Button from 'primevue/button';
-import Column from 'primevue/column';
-import DataTable from 'primevue/datatable';
-import Dialog from 'primevue/dialog';
-import ProgressSpinner from 'primevue/progressspinner';
+import Button from "primevue/button";
+import Column from "primevue/column";
+import DataTable from "primevue/datatable";
+import Dialog from "primevue/dialog";
+import ProgressSpinner from "primevue/progressspinner";
 
 // Other imports
-import axios from 'axios';
-import { useToast } from 'vue-toastification';
+import axios from "axios";
 
 // Other components
-import CreateContact from './CreateContact.vue';
+import CreateContact from "./CreateContact.vue";
 
 // Store
-const store: any = inject('store');
+const store: any = inject("store");
 
 // ----------------------------------------------------------------
 // Loading contacts
 // ----------------------------------------------------------------
 let loading = ref(true);
 
+const onRowSelect = (e: any) => {
+  console.log("onRowSelect: ", Object.keys(e.data));
+  console.log("onRowSelect: ", Object.values(e.data));
+  // displayAddContact.value = true;
+};
+
```

Comment:
Needed here if it's just console logging?

Reply:
Right... this was exploratory for another feature (delete row). Will remove for this PR.